setup.py

```diff
@@ -24,7 +24,8 @@
     "autograd",
     "toml",
     "appdirs",
-    "semantic_version==2.6"
+    "semantic_version==2.6",
+    "tensornetwork"
```

Comment:
```suggestion
    "tensornetwork"==0.1.1
```
for now, this is the only version we've tested against

Reply:
This will probably break CI though, due to some python3.5 issues
@josh146

Reply:
Yep, it is deliberate to leave it unpinned for now. This way it will install on 3.5, but the tests are just set to skip for tensornetwork <0.1

Reply:
Let's leave it out for now, until `expt.tensornet` is less experimental.

Reply:
```suggestion
```